src/download/linux.md

```diff
@@ -122,24 +122,63 @@ una install prismlauncher-git
 You can replace una install with your preferred [MPR helper's](https://docs.makedeb.org/using-the-mpr/list-of-mpr-helpers/) install command.
 </div>
 
-<!--
 <div class="infobox top">
 
 # <img src="https://www.vectorlogo.zone/logos/getfedora/getfedora-icon.svg" height="20"> Fedora
 
-Two RPM packages are available on [Copr](https://copr.fedorainfracloud.org/coprs/sentry/prismlauncher/).
+One RPM package is available on [Copr](https://copr.fedorainfracloud.org/coprs/g3tchoo/prismlauncher/).
```

Comment:
maybe change this to

> RPM packages are available

since there's one for all currently supported releases (plus it looks a little nicer imo)